File: api-cmssignature/api-mssclient/mssclient/src/tr/gov/tubitak/uekae/esya/api/webservice/mssclient/transaction/signature/ISignatureResponse.java
package tr.gov.tubitak.uekae.esya.api.webservice.mssclient.transaction.signature;

import tr.gov.tubitak.uekae.esya.api.common.ESYAException;
import tr.gov.tubitak.uekae.esya.api.webservice.mssclient.transaction.common.IResponse;

/**
 *Generic Mobile Signature response
 */
public interface ISignatureResponse extends IResponse {
    String getTransId();

    String getMSISDN();

    byte[] getSignature();

    byte[] getRawSignature() throws ESYAException;
}

File: api-cmssignature/api-mssclient/mssclient/csharp/src/wrapper/find/transaction/signature/ISignatureResponse.cs
using System;
using tr.gov.tubitak.uekae.esya.api.webservice.mssclient.transaction.common;

namespace tr.gov.tubitak.uekae.esya.api.webservice.mssclient.transaction.signature
{
    /**
 *Generic Mobile Signature response
 */

    public interface ISignatureResponse : IResponse
    {
        String getTransId();
        String getMSISDN();
        byte[] getSignature();
        byte[] getRawSignature();
    }
}